pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+name = "librepcb-parts-generator"
+authors = [
+    { name = "Danilo Bargen", email = "mail@dbrgn.ch" },
+    { name = "Hannes Badertscher", email = "h_badertscher@bluewin.ch" },
+    { name = "Murray", email = "murray.calavera@protonmail.com" },
+    { name = "ouabache", email = "z3qmtr45@gmail.com" },
+    { name = "Raphael Nestler", email = "raphael.nestler@gmail.com" },
+    { name = "Tubbles", email = "jae91m@gmail.com" },
+    { name = "U. Bruhin", email = "urbibruhin@bluewin.ch" },
```

Comment:
I'll remove most contributors and just leave @dbrgn and my as the main authors. U. Bruhin explicitly requested to be removed.

If anyone wants to add themself, they can just do it in a PR and we'll gladly merge it.